README.md

```diff
@@ -0,0 +1,57 @@
+Discovery API
+=============
+
+* [What is the status of this document?][statuses]
+* [See the index of all other EWP Specifications][develhub]
+
+
+Summary
+-------
+
+**Discovery API** is the first API that the partner developer needs to
+implement in order to become a basic member of the *Erasmus Without Paper*
+Network.
+
+ * It serves to identify you, announce which HEIs your system covers, and which
+   certificates you are going to use when you will be fetching the data from
+   the EWP Network. It's like a simple "business card" of your EWP Host.
+
+ * It serves to inform all **other** members of the network about which
+   features (APIs) **you** have implemented. This list MAY include external
+   APIs (unrelated to the EWP Project).
+
+
+Requirements
+------------
+
+In order to implement the Discovery API, you will need to:
+
+ * Host the Discovery Manifest file (described below) somewhere on your
+   servers.
+
+ * You MUST host it over HTTPS. The SSL certificate used MUST match your domain
+   and be signed by a trusted CA.
+
```

Comment:
Given the state of the art, I'd add that the server name SHOULD be published form a DNSSec signed zone (I use SHOULD, to allow some slack for those not able to get signed DNS zones, that should be fewer and fewer in HEI space) and maybe with the certificates pinned to DANE DNS records.